app/uk/gov/hmrc/tai/model/TaxCodeMismatch.scala

```diff
@@ -22,5 +22,13 @@ import play.api.libs.json.Json
 case class TaxCodeMismatch(mismatch: Boolean, unconfirmedTaxCodes: Seq[String], confirmedTaxCodes: Seq[String])
```

Comment:
Is the case class required?

Reply:
Not sure what you mean? Have addressed the other comments